java-client/src/main/java/com/cumulocity/sdk/client/RestConnector.java

```diff
@@ -114,8 +116,19 @@ public boolean isClosed() {
 
     @Override
     public <T extends ResourceRepresentation> T get(String path, CumulocityMediaType mediaType, Class<T> responseType) throws SDKException {
-        Response response = getClientResponse(path, mediaType);
-        return responseParser.parse(response, responseType, OK.getStatusCode());
+        try {
+            Response response = getClientResponse(path, mediaType);
+            return responseParser.parse(response, responseType, OK.getStatusCode());
+        } catch (SDKException e) {
+            logTenantUserMediaParameter(mediaType);
+            throw e;
+        }
```

Comment:
@hanneshofmann .
Oh I missed to say, that the response is already returned, if parseable
example 

`2022-05-19 07:14:48.213 ERROR 13 --- [subscriptions-0] c.cumulocity.sdk.client.ResponseParser   : Failed to parse error message to json. Getting error string... 
2022-05-19 07:14:48.214 ERROR 13 --- [subscriptions-0] c.cumulocity.sdk.client.ResponseParser   : <html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>openresty</center>
</body>
</html>
`
What I miss in case of Exception, what was trying to sent to the server, what could help for 300 or 400 errors.

Reply:
Ah, I see - thanks for the clarification. 👍 

I assume it would help to include the headers of the response. At least this would have helped us in one of the recent problems to identify which server has actually sent the invalid 503 response.